plugins/commando.c

```diff
@@ -428,6 +428,15 @@ static const char *check_condition(const tal_t *ctx,
 	if (!ptok)
 		return rune_alt_single_missing(ctx, alt);
 
+	/* Special case named `*_msat` fields */
+	if (strstarts(alt->fieldname, "pname")
```

Comment:
afaict the problem isn't the param's type, it's that we use `rune_alt_single_str` as the check/processor for every param type arg in `commando`; the real magic here is switching to `rune_alt_single_int`

Reply:
actually moving to "always ints" is really dope since i was slightly concerned about having to handle string-type expressions (hence the usage of `parse_msat`)